src/cdk-experimental/combobox/combobox.ts

```diff
@@ -74,9 +75,15 @@ export class CdkCombobox<T = unknown> implements OnDestroy, AfterContentInit {
   }
   private _openActions: OpenAction[] = ['click'];
 
+  @Input()
+  get autoSetText(): boolean { return this._autoSetText; }
+  set autoSetText(value: boolean) { this._autoSetText = coerceBooleanProperty(value); }
+  private _autoSetText: boolean = true;
```

Comment:
Add a JsDoc description that explains what this property does?

Reply:
Yeah will do.